src/main/io/beeper.c

```diff
@@ -55,13 +55,18 @@ PG_REGISTER_WITH_RESET_TEMPLATE(beeperDevConfig_t, beeperDevConfig, PG_BEEPER_DE
 #endif
 #ifdef BEEPER
 #define BEEPER_PIN      BEEPER
+#define BEEPER_PWM_FREQ 0
+#elif defined(BEEPER_PWM)
+#define BEEPER_PIN      BEEPER_PWM
```

Comment:
@ledvinap: Yes, what you suggest works too. I still prefer my solution (having only one define for the beeper pin, and another define to indicate the type. Since having more than one beeper pin defined for a board does not make much sense, having only one define for the pin eliminates having to cater for multiple pins being defined.
I see this as a valuable addition, and the option to use `frequency` to switch between on/off and PWM type as material for follow up PR, ideally combined with the ability for users to move the beeper resource between pins.